test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -22886,16 +22886,19 @@
           "IsNullable": false
         },
         {
-          "Name": "TermSeparators",
-          "Type": "String",
-          "Desc": "Comma separated set of term separator(s). Commonly: 'space', 'comma', 'semicolon' or other single character.",
+          "Name": "CharArrayTermSeparators",
+          "Type": {
+            "Kind": "Array",
+            "ItemType": "Char"
+          },
+          "Desc": "Array of single character term separator(s). By default uses space character separator.",
```

Comment:
>By default uses space character separator." [](start = 64, length = 43)

this and the current default are contradictory

Reply:
See other comment.

---
In reply to: [222366500](https://github.com/dotnet/machinelearning/pull/1089#discussion_r222366500) [](ancestors = 222366500)